sessionlockev/src/lib.rs

```diff
@@ -1048,15 +1119,6 @@ impl<T> Dispatch<wl_pointer::WlPointer, ()> for WindowState<T> {
                     },
                 ));
             }
-            wl_pointer::Event::Leave { .. } => {
-                state.current_surface = None;
-                state
-                    .message
-                    .push((state.surface_id(), DispatchMessageInner::MouseLeave));
-                if let Some(keyboard_state) = state.keyboard_state.as_mut() {
-                    keyboard_state.current_repeat = None;
```

Comment:
if this part is not been produced.. I am afraid that the keyboard repeating will be of bugs..

Reply:
I see, you move it above.. sorry for that